CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Removed
 
 ### Fixed
+- Fix Concurrent Modification Exception in Indices Request Cache([#14032](https://github.com/opensearch-project/OpenSearch/pull/14221))
```

Comment:
I see this PR is tagged with 2.15. If you've gotten consensus to get this into the on-going release then you should put this in the release notes file and not in the changelog.

Reply:
Good to know, thanks !

Reply:
Pushed a commit to address this, I guess i should add a label to skip change log ?